Apps/Common/src/Services/EmailQueueService.cs

```diff
@@ -192,6 +192,7 @@ private Email ParseTemplate(EmailTemplate emailTemplate, Dictionary<string, stri
             }
 
             Email email = new Email();
+            email.Id = Guid.NewGuid();
```

Comment:
For a new email, the database will generate the guid.

Reply:
this was a problem when saving the the message verification, it threw an error saying the emailId cannot be null for Email verification type.